### PR TITLE
Added clarification on documentation related to beats and seconds

### PR DESCRIPTION
**Pull request by parulpriyedarshani**
Solved issue #400 
Added proper clarifications in the documentation related to beats and seconds.
Areas of impact : Documentation

### CODE REVIEW

**Comment by parulpriyedarshani** [comment]
@therewasaguy Kindly, please review my PR.

**Comment by parulpriyedarshani** [comment]
@therewasaguy thanks for your review :)
apologies for the indentation error, it might have happened by mistake, I'm working on it as per your advice!

**Comment by parulpriyedarshani** [comment]
@therewasaguy As per your review, all the suggested changes have been made in the existing PR. Kindly,  please give it a look.  
Thanks

File: src/soundLoop.js [hunk 1]
```diff
@@ -6,12 +6,17 @@ define(function (require) {
 
   /**
    * SoundLoop
-   *
+   * Beats are the periodic and repeating fluctuations heard in the intensity of a sound when two sound waves of very similar frequency interfere with one another.
+   * Nodes are the locations where no sound is produced as a result of destructive interference.
+   * Beat pattern is characterized by a wave whose amplitude is changing at a regular rate.
```
Comment (therewasaguy) [inline src/soundLoop.js:11]: this is describing a different type of beat pattern that has to do with sound waves. We're talking about a step sequencer pattern, something like this https://drumbit.app/

**Comment by therewasaguy** [comment]
closing for now, feel free to reopen if you update the branch and thank you @parulpriyedarshani

**Pull request closed without merging**